index.html

```diff
@@ -565,19 +677,31 @@ <h4>Script Type</h4>
                 <p>It can be adapted to produce a <a>Pre-Recording Script</a>,
                   and/or used as the basis for a further translation into the <a>Target Recording Language</a>.</p> 
                 <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Translation</a>.
-                  They MAY also contain <a>Original</a> language <a>Text</a>.</p>
+                  <a>Text Language Source</a> is set to the source language from which they were translated,
+                  and whose language is set to their actual language,
```

Comment:
I see you rephrased differently. Upon reading again, I wonder if we should not simply remove the part 
> marked as such by having their Text Language Source set to the source language from which they were translated, and their language set to their actual language

If "Translation Text" links to a clear specification of how to signal it there is no need to repeat it here (and elsewhere).

Reply:
I would suggestion using the following:

In "Original Language Transcript":
> Script Events in this type of transcript SHOULD contain Original Text objects and SHOULD NOT contain Translation Text objects.

and:
> EXAMPLE 13
> If a programme contains dialogue in English and Hebrew, the Original Language Transcript will contain some Script Events in English and some in Hebrew, all containing Original Text. None of the Script Events has Translation Text.

In "Translated Transcript":
> Script Events in this type of transcript SHOULD contain Translation Text objects. They MAY also contain Original Text objects.

> EXAMPLE 15
If an audio description Original Language Transcript contains Original language Script Events that describe in-image text content, and the desired audio description output needs to be in a different language, the Text objects can be Translation Text objects. In that case, it is appropriate to mark the transcript as a Translated Transcript. It may also be appropriate, as part of the translation activity, to describe within the translated text the original language of that text, to inform the audience.

In "Pre-recording script":
> NOTE
The Script Type of a DAPT Script cannot necessarily be detected by inspecting the text content of the document. For example, the adaptation of a Translated Transcript into a Pre-recording Script can consist in replacing some words in the text content of a Script Event without altering the rest of the document. In either case, Translation Text objects have their Text Language Source properties set to the source language from which they were translated.

> NOTE
The Original Text objects in Audio Description Script Events do not have a Text Language Source property if they represent visual elements of the scene that do not have an inherent language, but are required to have a Text Language Source if they do represent visual elements with an inherent language, such as in-image text. If audio description scripts are translated, their translations would be represented by Translation Text objects.

Reply:
Done in fa7ca74, though the last proposal was slightly overtaken by other changes. However the important part seemed to be the final sentence, for which I made the equivalent change.